### DISCUSSION TITLE
Figure out If user selected folder or file

### DISCUSSION THREAD

**Discussion by arjar88** [react-dropzone/react-dropzone#1351] [Q&A] [answered]
Hey all,
I wanted to know if there is a way to tell if the content that is being selected is a folder or a file?
Right now when I select a folder ,all of the contained files will be passed in the `acceptedFiles` props but know way to know if a folder was selected.
thank you.

**Comment by rolandjitsu** [comment, marked as answer]
@arjar88 the library doesn't specifically tell you, but the easiest way I can think of is to check if the files have a directory in the path or not.

When you drop a directory, the path should look something like `dir/image.png`, whereas when you drop a file, the path should be `image.png`.
